.editorconfig

```diff
@@ -10,110 +10,140 @@ root = true
 insert_final_newline = true
 indent_style = space
 indent_size = 4
+trim_trailing_whitespace = true
 
 [project.json]
 indent_size = 2
 
+# Generated code
+[*{_AssemblyInfo.cs,.notsupported.cs}]
```

Comment:
Do we need to include `.Generated.cs`? For example: https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Model/V1.Generated.cs

Reply:
Good catch